examples/nidmm/continuous-acquire-multiple-points.py

```diff
@@ -0,0 +1,186 @@
+#
+# The gRPC API is built from the C API.  NI-DMM documentation is installed with the driver at:
+# C:\Program Files (x86)\IVI Foundation\IVI\Drivers\niDMM\Documentation\English\DMM.chm
+
+#
+# Getting Started:
+#
+# To run this example, install "NI-DMM Driver" on the server machine.
+# Link : https://www.ni.com/en-us/support/downloads/drivers/download.ni-dmm.html
+#
+# For instructions on how to use protoc to generate gRPC client interfaces, see our "Creating a gRPC Client" wiki page.
+# Link: https://github.com/ni/grpc-device/wiki/Creating-a-gRPC-Client
+#
+# Refer to the NI-DMM gRPC Wiki to determine the valid channel and resource names for your NI-SCOPE module.
+# Link : https://github.com/ni/grpc-device/wiki/NI-DMM-C-Function-Reference
+#
+# Running from command line:
+#
+# Server machine's IP address, port number, and resource name can be passed as separate command line arguments.
+#   > python continuous-acquire-multiple-points.py <server_address> <port_number> <resource_name>
+# If they are not passed in as command line arguments, then by default the server address will be "localhost:31763", with "SimulatedDMM" as the resource name
+
+import grpc
+import sys
+import time
+import nidmm_pb2 as nidmm_types
+import nidmm_pb2_grpc as grpc_nidmm
+import session_pb2 as session_types
+import session_pb2_grpc as grpc_session
+
+server_address = "localhost"
+server_port = "31763"
+session_name = "NI-DMM-Session-1"
+
+# Resource name and options for a simulated 4065 client. Change them according to the NI-DMM model.
+resource = "SimulatedDMM"
+options = "Simulate=1, DriverSetup=Model:4065; BoardType:PCI"
+
+# parameters
+MAXPTSTOREAD        = 1000
+config_range        = 10.00
+resolution          = 5.5
+measurementType     = nidmm_types.Function.FUNCTION_NIDMM_VAL_DC_VOLTS
+
+# Read in cmd args
+if len(sys.argv) >= 2:
+    server_address = sys.argv[1]
+if len(sys.argv) >= 3:
+    server_port = sys.argv[2]
+if len(sys.argv) >= 4:
+    resource = sys.argv[3]
+    options = ""
+
+# Create the communication channel for the remote host and create connections to the NI-DMM and session services.
+channel = grpc.insecure_channel(f"{server_address}:{server_port}")
+nidmm_client = grpc_nidmm.NiDMMStub(channel)
+
+any_error = False
+# Checks for errors. If any, throws an exception to stop the execution.
+def CheckForError (vi, status) :
+    global any_error
+    if(status != 0 and not any_error):
+        any_error = True
+        ThrowOnError (vi, status)
+
+# Converts an error code returned by NI-DMM into a user-readable string.
+def ThrowOnError (vi, error_code):
+    error_message_request = nidmm_types.ErrorMessageRequest(
+        vi = vi,
+        error_code = error_code
+    )
+    error_message_response = nidmm_client.ErrorMessage(error_message_request)
+    raise Exception (error_message_response.error_message)
+
+try:
+    # Open session to NI-DMM with options
+    print('\nInitializing session...')
```

Comment:
no need to print messages twice - Initializing, Initialized, configuring, configured etc. 
it will be almost immediate & would not add any value. In fact in the DMM example we may not need to print these messages at all. (In switch example we print those messages to indicate that the example is indeed running as there is no output as such.)

Reply:
removed prints